support/scripts/uk-libinfo.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+# Licensed under the BSD-3-Clause License (the "License").
+# You may not use this file except in compliance with the License.
+
+# NOTE: The script requires pyelftools (pip3 install)
+import os
+import sys
+import argparse
+from elftools.elf.elffile import ELFFile
+
+class UKLibraryInfoRecord:
+	def __init__(self, type, data,  sizeof_ushort, sizeof_uint, byteorder):
+		self._tnum  = int(type)
+		self._tname = None
+		self._tdesc = None
+		self._data  = None # Tuple: (is_raw, decoded_data)
+		match int(self._tnum):
+			#
+			# NOTE: This list needs to kept in sync with
+			#       lib/uklibid/include/uk/libinfo.h
+			#
+			case 0x0001:
+				self._tname = "LIBNAME"
+				self._tdesc = "Library name"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0002:
+				self._tname = "COMMENT"
+				self._tdesc = "Comment"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0003:
+				self._tname = "VERSION"
+				self._tdesc = "Version"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0004:
+				self._tname = "LICENSE"
+				self._tdesc = "License"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0005:
+				self._tname = "GITDESC"
+				self._tdesc = "Git version"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0006:
+				self._tname = "UKVERSION"
+				self._tdesc = "Unikraft version"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0007:
+				self._tname = "UKFULLVERSION"
+				self._tdesc = "Unikraft full version"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0008:
+				self._tname = "UKCODENAME"
+				self._tdesc = "Unikraft codename"
+				self._data  = self._data_decode(data, 'ascii')
+			case 0x0009:
+				self._tname = "UKCONFIG"
+				self._tdesc = "Unikraft .config"
+				self._data  = (True, data)
```

Comment:
I agree, it was also the plan. My goal was to have thi early version as part of this release so that we have the infrastructure and can add these missing items to the tool or build system afterwards.

Reply:
Ok, we can do it when it's needed then 👍🏼